test/tests/functional/pbs_qrun.py

```diff
@@ -129,6 +129,7 @@ def test_qrun_hangs(self):
             try:
                 self.server.runjob(jobid=jid)
                 self.logger.info("Successfully runjob. Child process exit.")
+                os._exit(0)
```

Comment:
Just say 'return' ? \_exit sounds like an internal function since it starts with an underscore

Reply:
I had tried return but that does not terminate the current process right. With return, I still see 2 test complete results being generated.

Reply:
ah yes, return will cause it to go through the same post run functions that the normal process goes through. I still don't like the idea of using an internal function because that can get renamed one fine day. How about just doing "sys.exit(0)" ?

Reply:
@agrawalravi90 `os._exit(0)` is not an internal method. it is exposed a valid method that anyone can use it. And python doc itself say child should use `os._exit(0)`. See https://docs.python.org/3.8/library/os.html#os._exit

Reply:
I do not think this is something internal. It is a documented API and it is not easy to change it's name.
Also, from the [documentation](https://docs.python.org/3/library/os.html#os._exit)
_exit() should normally only be used in the child process after a fork().

Reply:
Ok, weird that they underscore it then, thanks for the pointer